lua-libraries/qrencode.lua

```diff
@@ -175,15 +175,14 @@ end
 -- See table 2 of the spec. We only support mode 1, 2 and 4.
 -- That is: numeric, alaphnumeric and binary.
 local function get_mode( str )
-	local mode
 	if string.match(str,"^[0-9]+$") then
 		return 1
 	elseif string.match(str,"^[0-9A-Z $%%*./:+-]+$") then
 		return 2
 	else
 		return 4
 	end
-	assert(false,"never reached")
+	assert(false,"never reached") -- luacheck: ignore
```

Comment:
Is this really a vendored library or something you worked up?

Reply:
Vendored from https://github.com/speedata/luaqrcode (which doesn't have a rockspec).

Just temporarily enabling luacheck on it, and upgrading to their latest version (with linting commits from me, the author etc.)

Reply:
Gotcha. You know you can lint locally for temporary answers too ;-) No problems doing it in CI I guess, just faster to do locally especially for one-off answers to questions outside of the remote configuration.